programme/validate-palindrome/validate-palindrome.java

```diff
@@ -1,6 +1,6 @@
 class Main {
     public static void main(String[] args) {
-        String str1 = "level";
+        String str1 = "radar";
```

Comment:
In the future, if I commit again, should I just always use the input that is stated in the issue?

I used "level" here because that is the string used in the example in the README.

Reply:
Oh sorry, I didn't see that. I guess it should be "level" because of the readme. In the future also you can follow that. Also add ur name as a contributor in readme file.